qabot/pipeline_maintenance.py

```diff
@@ -229,14 +230,46 @@ def ci_benchmarking(self, repo_name, pr_num, stage_name):
 
         return bot_response
 
+    def create_ticket(self, type, args=""):
```

Comment:
maybe this should be in a different module? pipeline_maintenance seems more like jenkins tasks

Reply:
I guess the original rationale was that.. whenever a CI cycle that is running on the pipeline fails, people can leverage this automation to create tickets.

This should be used by Thor to automatically cut bug tickets to SMEs whenever a given release test cycle fails.
🤷🏼‍♂️